scripts/download_blenderkit.py

```diff
@@ -10,68 +10,74 @@
 from pathlib import Path
 import uuid
 
-assets = []
-page = 1
-while True:
-    print("Download metadata: page {}".format(page))
+asset_types=["material","model"]
+assets = {}
 
-    # Download one page of metadata
-    try:
-        with urllib.request.urlopen("https://www.blenderkit.com/api/v1/search/?query=asset_type:material+order:_score+is_free:True&addon_version=1.0.30&page=" + str(page)) as url:
-            data = json.loads(url.read().decode())
-            # Extract results
-            assets.extend(data["results"])
-    except HTTPError as e:
-        if e.code == 404:
-            # We reached the end
-            break
-        else:
-            raise
-    # Goto next page
-    page += 1
+for asset_type in asset_types:
+    page = 1
+    max_pages = 10
+    while page < max_pages:
```

Comment:
This should be, the usage of an external variable is not PEP-8 conform: 
```suggestion
    for page in range(max_pages):
```